cPP_DSC.adoc

```diff
@@ -190,7 +190,7 @@ DSCs provide seven core security services to a platform as illustrated in <<Core
 |Description
 
 |Parse
-|The DSC shall ingest pre-provisioned keys, credentials, tokens, attributes, etc. from trusted components or services external to its boundary either across a secured channel or in a manner that the objects are protected for use only by the DSC.
+|The DSC shall ingest pre-installed keys, credentials, tokens, attributes, etc. from trusted components or services external to its boundary either across a secured channel or in a manner that the objects are protected for use only by the DSC.
```

Comment:
This is one of the only two occurences of "pre-provisioned" in the document; everywhere else "pre-installed" is used. I suggest changing it here for consistency.